doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -607,344 +797,50 @@ Response code:
 
 + Response 204
 
+## Standard queryContext [/v2/queryContext]
 
-## Context Entity Subscriptions [/v2/entities/{entityId}/subscriptions]
-
-### List all subscriptions present in the system [GET /v2/subscriptions]
-
-+ Response 200
-
-        [
-            {
-                "id": "abcdefg",
-                "entity": {
-                    "id": "Bcn_Welt",
-                    "type": "Room"
-                },
-                "attributes": [
-                    "temperature"
-                ],
-                "callback": "http://localhost:1234",
-                "options": {
-                    "duration": "1M",
-                    "condition": "ONCHANGE",
-                    "throttling": "PT5S"
-                }
-            },
-             {
-                "id": "wyztrew",
-                "entity": {
-                    "pattern": "DC_*",
-                    "type": "Room"
-                },
-                "attributes": [
-                    "temperature"
-                ],
-                "callback": "http://localhost:1234"
-            }
-        ]
-
-### Get the details of a subscription [GET /v2/subscriptions/{subscriptionId}]
-
-+ Parameters
-    + subscriptionId: abcdefg (required, string) - ID of the suscription
-
-+ Response 200
-
-        {
-            "id": "abcdefg",
-            "entity": {
-                "id": "Bcn_Welt",
-                "type": "Room"
-            },
-            "attributes": [
-                "temperature"
-            ],
-            "callback": "http://localhost:1234",
-            "options": {
-                "duration": "1M",
-                "condition": "ONCHANGE",
-                "throttling": "PT5S"
-            }
-        }
-
-### Remove a suscription [DELETE /v2/subscriptions/{subscriptionId}]
-
-+ Parameters
-    + subscriptionId: abcdefg (required, string) - ID of the suscription
-
-+ Response 204
-
-
-### Create a new subscription [POST /v2/subscriptions]
+### queryContext [POST /v2/queryContest]
 
-+ Request (application/json)
+## Standard updateContext [/v2/updateContext]
```

Comment:
I have created some placeholders for standard operations. However, I don't like too much the result: by the way of working of Apiary, given that each std op uses a different URL, we have to created a lot of 2-level sections which only containg one 3-level section each one (the corresponding to POST). The result is a left index very large (it would be great to have just one item in that index, all the std ops grouped together there).

Not sure if what I want to achieve is not possible in APIary...